api/cpp/include/slint.h

```diff
@@ -185,16 +185,26 @@ class WindowAdapterRc
         return pos;
     }
 
-    void set_position(const slint::Point<int> &pos)
+    void set_logical_position(const slint::Point<float> &pos)
     {
-        cbindgen_private::slint_windowrc_set_position(&inner, &pos);
+        cbindgen_private::slint_windowrc_set_logical_position(&inner, &pos);
+    }
+
+    void set_physical_position(const slint::Point<int> &pos)
+    {
+        cbindgen_private::slint_windowrc_set_physical_position(&inner, &pos);
     }
 
     slint::Size<unsigned int> size() const { return cbindgen_private::slint_windowrc_size(&inner); }
 
-    void set_size(const slint::Size<unsigned int> &size)
+    void set_logical_size(const slint::Size<float> &size)
```

Comment:
In the C++ API:

Should we not also have a `slint::PhysicalSize`/`LogicalSize` types?

And then, should we just have a set_size with a  `std::variant<PhysicalSize, LogicalSize>` ?

Reply:
Yes, that would be modern, but std::variant is so inconvenient to use!

Reply:
Can you elaborate why it is incontinent?  Are you referring to the visit function?
In this case, we only want to use that as an input parameter, so the fact that the `PhysicalSize` and logical one would just convert should be quite convenient.

Reply:
I added `(Logical|Physical)(Position|Size)` to the relevant headers and used those throughout.

The `set_size` and `set_position` methods are both overloaded to take both types, so I see no need for a `std::variant`.

Reply:
Ah yes, i forgot about overloads :-P